environment/src/main/java/com/sequenceiq/environment/experience/liftie/responses/ClusterView.java

```diff
@@ -0,0 +1,92 @@
+package com.sequenceiq.environment.experience.liftie.responses;
+
+import java.util.StringJoiner;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.swagger.annotations.ApiModel;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@ApiModel(value = "ClusterView")
+public class ClusterView {
```

Comment:
We already have `ClusterView` class in our code base, I think this should be a Liftie specific cluster view, which could be `LiftieClutserView`. As we already referencing to the appropriate `ClusterView` model object with `@ApiModel` annotation.

Reply:
renamed and also removed the `@ApiModel` annotation as we are not publishing these objects as part of any REST interface from our own services